plugins/redacted/tracker.py

```diff
@@ -152,3 +152,7 @@ def get_download_location_context(self, torrent_info):
             }
         except RedactedTorrent.DoesNotExist:
             return {}
+
+    def get_snatched(self, offset, limit):
```

Comment:
Can you please remove that from the tracker for now? Can the manage.py command use the client directly?

Reply:
Actually, scratch that. Maybe it's OK here...